include/odp/api/spec/packet_io.h

```diff
@@ -1394,6 +1397,69 @@ uint64_t odp_pktin_ts_res(odp_pktio_t pktio);
  */
 odp_time_t odp_pktin_ts_from_ns(odp_pktio_t pktio, uint64_t ns);
 
+/**
+ * Packet IO timestamp resolution in hertz
+ *
+ * This is the resolution of packet input and packet output timestamps.
+ * using Packet IO time source. Returns zero on a failure or when both
+ * packet in and packet out timestamping is disabled.
+ *
+ * @param      pktio   Packet IO handle
+ *
+ * @return Packet IO timestamp resolution in hertz
+ * @retval 0 on failure
+ */
+uint64_t odp_pktio_ts_res(odp_pktio_t pktio);
+
+/**
+ * Convert nanoseconds to Packet IO time
+ *
+ * Packet IO time source is used for timestamping incoming packets and outgoing
+ * packets. This function is used convert nanosecond time to
+ * packet input or packet output timestamp time.
+ *
+ * @param      pktio   Packet IO handle
+ * @param      ns      Time in nanoseconds
+ *
+ * @return Packet IO timestamp
+ */
+odp_time_t odp_pktio_ts_from_ns(odp_pktio_t pktio, uint64_t ns);
+
+/**
+ * Current Packet IO time and global time
+ *
+ * Returns current Packet IO time and optionally
+ * global time. This global time is that of global time source
+ * where as Packet IO timestamp is of Packet IO time source that
+ * is used to timestamp incoming and outgoing packets.
+ *
+ * @param      pktio        Packet IO handle
+ * @param      ts_global    Pointer to odp_time_t.
+ *                          When not NULL, on success, global timestamp
+ *			    will be updated taken at same point of time
+ *			    as Packet IO time.
```

Comment:
Indent comment text using spaces only. Mix of tabs and spaces does not show clean on different editors/web view/etc.